doc/polyglossia.tex

```diff
@@ -5,16 +5,16 @@
 \usepackage{xspace,fancyvrb,longtable,booktabs}
 \usepackage[neverdecrease]{paralist}
 \usepackage[format=hang,labelfont=bf,labelsep=period]{caption}
-\definecolor{myblue}{rgb}{0.02,0.04,0.48}
+\definecolor{pgblue}{rgb}{0.02,0.04,0.48}
```

Comment:
for the prefixes, either use `xpg` or `polyglossia`, not a third variant.

Reply:
Wasn't it you, who introduced the `pghyphenation` command? ;-)
So far, no prefixes have been used in the manual (except “my” in `myblue` and `myred`, which seems inappropriate in a public project).
In the package code, there are three types of prefixes: `xpg@`, `polyglossia@`, and `polyglossia_`. Obviously, none of them can be used here as special characters are not allowed in document commands.
If you don't like `pg`, I'd rather suggest to renounce the prefix and to use `\boolkey` instead of `\pgboolkey` and so forth.

Reply:
I'd really prefer `xpgblue` etc.

Reply:
OK, as you like it.